cmd/baton-atlassian/main.go

```diff
@@ -60,7 +60,14 @@ func getConnector(ctx context.Context, v *viper.Viper) (types.ConnectorServer, e
 	organization := v.GetString(organizationField.FieldName)
 	siteId := v.GetString(siteIdField.FieldName)
 
-	connectorBuilder, err := connectorSchema.New(ctx, userEmail, apiToken, organization, siteId)
+	connectorBuilder, err := connectorSchema.New(
+		ctx,
+		userEmail,
+		apiToken,
+		fmt.Sprintf("ari:cloud:platform::org/%s", organization),
```

Comment:
I think a better change would be to have connector New() check whether organizationID starts with `ari:cloud:platform::org/` and if it doesn't, prefix the org ID with that string. That way we create the correct org ID even if the connector is instantiated some other way (such as in a test or if directly invoked by c1's temporal-worker). It would also make us forwards-compatible with any changes to Atlassian Resource Identifiers.